Week02/lesson04/homework/relaxr_starter_code/css/style.css

```diff
@@ -0,0 +1,200 @@
+/*
+Project Name: Relaxr Homepage
+Client: Relaxr
+Author: Christophe Brumby
+Developer @ GA in London
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 1% 1%; /* note the padding around a right floated image */
+  width: 50%;
+  text-align: left;
+}
+
+.alignleft {
+  float: left;
+  padding: 0 1% 1% 0; /* note the padding around a left floated image */
+  width: 50%;
+  text-align: right;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+     color: #000;
+     font-size: 12px;
+     line-height: 1.4;
+     font-family: Open Sans, Helvetica, Arial, sans-serif;
+     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
+      background-size: contain;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container {
+  width: auto;
+}
+
+header {
+  padding-bottom: 22%;
+}
+
+footer {
+  padding: 3% 0 1%;
+  background-color: #121212;
+}
+
+/******************************************
+/* ADDITIONAL STYLES
+/*******************************************/
+
+h1 {
+  font-weight: 800;
+  font-size: 24px;
+  color: #FFF;
+  margin: 1% 0 8% 6%;
+}
+
+h2 {
+  font-weight: 600;
+  font-size: 24px;
+  color: #FFF;
+  margin-bottom: 7%;
+  text-align: center;
+}
+
+h3 {
+  font-weight: 600;
+  font-size: 20px;
+  color: #033048;
+  margin: 0 0 1% 6%;
+}
+
+h4 {
+  font-weight: 800;
+  font-size: 24px;
+  color: #f9e42e;
+  padding: 3% 0 5%;
+  text-align: center;
+}
+
+#button1 {
+  font-weight: 800;
+  font-size: 14px;
+  color: #033048;
+  background-color: #f9e42e;
+  padding: 10px;
+  border: 1px solid #f9e42e;
+  border-radius: 5px;
+  width: 150px;
+  text-align: center;
+  margin: auto;
+}
+
+main {
+  background-color: #f0efef;
+  padding: 5% 0;
+}
+
+p, ul {
+  font-weight: 300;
+  font-size: 13px;
+  color: #606161;
+  margin: 0 40% 3% 6%;
+}
+
+ul {
+  display: block;
+  list-style-type: disc;
+  padding-left: 40px;
+  padding-right: 60px;
+}
+
+#testimonial {
+  color: #FFF;
+  text-align: center;
+  background: url(../images/testimonial_bg.jpg) repeat center top rgb(134,141,157);
```

Comment:
Don't forget to use `no-repeat` instead of `repeat` to avoid the tiling effect. It isn't noticeable on a smaller screen, but on a big screen look what happens:

![image](https://cloud.githubusercontent.com/assets/5670279/17461894/b44a12a4-5c92-11e6-87b8-5e4849c19d4e.png)

... of course you don't want to end up with that weird grey background colour on the sides of the background-image, so as we learned last week, that's where `background-size: cover;` comes in